ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedStream.java

```diff
@@ -61,13 +61,13 @@ public SchemaKTable aggregate(final Initializer initializer,
     final KTable aggKtable;
     if (windowExpression != null) {
       final Materialized<String, GenericRow, ?> materialized
-          = Materialized.<String, GenericRow, WindowStore<Bytes, byte[]>>as(storeName)
+          = Materialized.<String, GenericRow, WindowStore<String, GenericRow>>as(storeName)
```

Comment:
This is actually incorrect. It should be `WindowStore<Bytes, byte[]>` the underlying store is always of type `<Bytes, byte[]>` now.

It should be:
```
Materialized.<String, GenericRow, WindowStore<Bytes, byte[]>>as(storeName)
   .withKeySerde(Serdes.String())
   .withValueSerde(topicValueSerDe)

Reply:
@dguy thanks for the correction. Fixed it.